playbooks/ceph.yml

```diff
@@ -92,45 +92,6 @@
       cifmw_admin_distribute_private_key: true
       no_log: true
 
-- name: Create Block Device on target nodes
```

Comment:
I don't think the nodes in the `podified-multinode-hci-deployment-crc` job will get the extra disks, and thus we still need this role for that scenario.

I think I'll need to have a parameter that the unified jobs can pass so that the `cifmw_block_device` role isn't called and if that's the case, then the spec will just use all devices and we'll assume the devices will come from elsewhere.

No mention of `/dev/vd{b,c,d}` in the storage logs from the failing job.

https://logserver.rdoproject.org/50/2250/71192a64fba6152631de7cd267de29e9f7c5771b/github-check/podified-multinode-hci-deployment-crc/8f10dfd/controller/ci-framework-data/logs/38.102.83.159/log/anaconda/storage.log

https://logserver.rdoproject.org/50/2250/71192a64fba6152631de7cd267de29e9f7c5771b/github-check/podified-multinode-hci-deployment-crc/8f10dfd/controller/ci-framework-data/logs/38.102.83.197/log/anaconda/storage.log